modules/networking/main.tf

```diff
@@ -1,129 +1,128 @@
-data "aws_availability_zones" "available" {}
```

Comment:
there was a good way to get AZs, I'd offer to revert it

Reply:
Done.